offset.go

```diff
@@ -31,15 +31,19 @@ const (
 
 // MarshalText implements the encoding.TextMarshaler interface.
 func (off *Offset) MarshalText() ([]byte, error) {
-	if off.position == OffsetPositionStart || off.position == OffsetPositionEnd {
-		return []byte(off.position), nil
+	if off == nil {
```

Comment:
Is this possible?

Reply:
yeah its pointer. the xml package will basically call `nil.MarshalText()` on any Offset object regardless of if its a real object or not when its marshalling a vmap object